src/pages/VenueLandingPage/VenueLandingPage.tsx

```diff
@@ -155,267 +154,256 @@ export const VenueLandingPage: React.FunctionComponent<VenueLandingPageProps> =
   const isPasswordRequired = venue.access === VenueAccessMode.Password;
 
   return (
-    <WorldUsersProvider venueId={venueId}>
-      <WithNavigationBar>
-        <div className="container venue-entrance-experience-container">
-          <div
-            className="header"
-            style={{
-              background: `linear-gradient(
+    <WithNavigationBar>
```

Comment:
We could potentially/probably even roll `WithNavigationBar` into `Provided` I think.

Reply:
I removed the provider from `WithNavigationBar` and worked in most cases, but did trip some OOS bug https://github.com/sparkletown/sparkle/pull/1769/commits/aa35a4db8b00596596a6fb5b9e8edf3c0a1839ab
so I left it there, as an exception until another PR fixes that issue.

My guess is something to do with https://github.com/facebook/react/issues/18178 or, more specifically to the case, some child component should use `useEffect()` when passing event handler. Can't remember the specific names, but can be reproduced by following the comment in the code https://github.com/sparkletown/sparkle/commit/aa35a4db8b00596596a6fb5b9e8edf3c0a1839ab.

Reply:
See my response in https://github.com/sparkletown/sparkle/pull/1769#discussion_r667284025

But basically I believe it's in line with this:

> Some child component should use `useEffect()` when passing event handler.

Reply:
> We could potentially/probably even roll `WithNavigationBar` into Provided I think.

Calling this OOS for this PR. Might be something interesting to look at doing in future though.

Reply:
K, reason enough to resolve this convo.